packages/backend/src/core/FeaturedService.ts

```diff
@@ -88,9 +98,17 @@ export class FeaturedService {
 		await redisPipeline.exec();
 	}
 
+	// 30%の確率でランキングを更新する
```

Comment:
このPRで30%固定ではなくなるので、このコメントは不正確です。
```suggestion
	// 更新比率に応じてランキングを更新する
```
など?